### PR TITLE
Use ActiveJob instead of Resque directly [requires Rails 5.0]

### PR DESCRIPTION
**Pull request by paroga**
This is a newer and cleaner version of #749, which requires Rails 5.0.

### CODE REVIEW

File: app/jobs/application_job.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class ApplicationJob < ActiveJob::Base
+  def serialize
+    super.merge(foodcoop: FoodsoftConfig.scope)
+  end
+
+  def deserialize(job_data)
+    FoodsoftConfig.select_multifoodcoop job_data['foodcoop']
+    super(job_data)
+  end
```
Comment (wvengen) [inline app/jobs/application_job.rb:9]: Will this also work when running inline (without a background-job backend)?

Comment (paroga) [reply app/jobs/application_job.rb:9]: since it runs inline there is no need to switch the database :wink:

Comment (wvengen) [reply app/jobs/application_job.rb:9]: hah, you got me ;)

**Comment by paroga** [comment]
reverted, since this PR has problems with multicoop setup

**Comment by wvengen** [comment]
Ouch, thank you! I've added the label 'needs work' to reflect that :)

**Pull request merged**